src/test/kotlin/uk/gov/gdx/datashare/queue/AwsQueueFactoryTest.kt

```diff
@@ -215,10 +241,12 @@ class AwsQueueFactoryTest {
       val sqsProperties = SqsProperties(provider = "localstack", queues = mapOf("somequeueid" to someQueueConfig))
       awsQueues = awsQueueFactory.createAwsQueues(sqsProperties)
 
-      verify(sqsClient).createQueue(
-        check<CreateQueueRequest> {
-          assertThat(it.attributes()).containsEntry(QueueAttributeName.REDRIVE_POLICY, """{"deadLetterTargetArn":"some dlq arn","maxReceiveCount":"2"}""")
-        },
+      val createQueueRequest = mutableListOf<CreateQueueRequest>()
+      verify { sqsClient.createQueue(capture(createQueueRequest)) }
+
+      assertThat(createQueueRequest.last().attributes()).containsEntry(
```

Comment:
There's an odd interaction here that I think's worth calling out - the  `awsQueueFactory...` method gets called in a `BeforeEach`  as well as in this method, which means this gets called twice but we only care about the last one